### PR TITLE
#18 Editing export excel

### PR DESCRIPTION
**Pull request by ebausson-obeo**
* Adding 'situation' as the first colum header.
 * Repeting situation name in every column in 'OR' loop.
 * Removing conposant name from header (sm1 instead of LC1/sm1).
 * Removing date from filename, auto-delete old file when re-generating (objective is ease of versionning).

### CODE REVIEW

File: plugins/org.polarsys.capella.vp.ms.expression.ag/src/org/polarsys/capella/vp/ms/expression/ag/SituationExpressionExportHandler.java [hunk 1]
```diff
@@ -89,6 +83,7 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
     try {
       exporter.export(toExport, columnOrder, bytes);
       ByteArrayInputStream inbb = new ByteArrayInputStream(bytes.toByteArray());
+      file.delete(false, new NullProgressMonitor());
```
Comment (pdulth) [inline plugins/org.polarsys.capella.vp.ms.expression.ag/src/org/polarsys/capella/vp/ms/expression/ag/SituationExpressionExportHandler.java]: Not sure, but raise an CoreException when the file doesn't exist, which make the command doesn't work when the file doesnt exist) no?
maybe a if (file.exists())

**Review by pdulth — CHANGES_REQUESTED** [review]

File: plugins/org.polarsys.capella.vp.ms.expression.ag/src/org/polarsys/capella/vp/ms/expression/ag/SituationExpressionExportHandler.java (hunk 1 above)
Comment (ebausson-obeo) [reply plugins/org.polarsys.capella.vp.ms.expression.ag/src/org/polarsys/capella/vp/ms/expression/ag/SituationExpressionExportHandler.java]: Fixed.
Also renamed commit to reference #issue.

**Review by lredor — COMMENTED** [review]
Before merging some remarks:
* Typo in commit message `Adding 'situation' as the first colum header.` --> column
* Typo in commit message `Removing conposant name from header (sm1 instead of LC1/sm1).`--> composant
 * Copyright to bump from 2023 instead of 2024

**Comment by ebausson-obeo** [comment]
Fixed.
Also corrected typo in 'versioning'.

**Pull request merged**